docs/UserGuide.md

```diff
@@ -128,26 +128,31 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting a student / task : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified student or task from their respective lists.
 
-Format: `delete INDEX`
+* Delete a student at specified index
+  - Format: `delete s/STUDENT_INDEX`
+* Delete a task at specified index
+  - Format: `delete t/TASK_INDEX`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
+**Things to Note:**
+* The index refers to the index number shown in the displayed list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `list s` followed by `delete s/2` deletes the 2nd student in Jarvis.
+* `list t` followed by `delete t/1` deletes the 1st task in Jarvis.
```

Comment:
i think should refer to JARVIS in all uppercase to be consistent

Reply:
ah yes my bad thx!